app/utilities/Iframe.tsx

```diff
@@ -64,6 +64,17 @@ const Iframe = ({ children, ...props }: { children: ReactNode }) => {
         style.appendChild(iframeDoc.createTextNode(customStyles));
         iframeDoc.head.appendChild(style);
 
+        iframeDoc.body.classList.add(
```

Comment:
Why did you add this?

Reply:
To center the the component in the preview body.